Makefile

```diff
@@ -1,3 +1,3 @@
 test:
-	@flake8 . --exclude fabfile.py --ignore=E241,E501,F403 # FIXME: Fix these errors and stop ignoring them
+	@flake8 . --exclude fabfile.py,venv --ignore=E241,E501,F403 # FIXME: Fix these errors and stop ignoring them
```

Comment:
Bit of a nitpick, but I'm not sure I agree with including a virtualenv directory here. Not everyone has their virtualenv located in the same place.